### ISSUE TITLE
XML format for Vikazimut

### ISSUE THREAD

**Issue by Eric-P-Kmz** [OpenOrienteering/mapper#2166]
### Steps to reproduce
We have continued to improve the link between OOM and Vikazimut for creating routes from OOM.
1 - We've managed the differences between the IOF and OOM formats in the XML file, but the map scale is missing from your XML.

 2 - Would it be possible to create multiple races in one XML?

 3 -  If you can make it easier to create the definition table, it would remove a big drawback to creating courses under OOM.

### Actual behaviour
2 - Only one race in XML

3 - No GUI to build the definition table

### Expected behaviour
1 - Is it possible to add this tag to the XML?
RaceCourseData>
Map>
Scale>10000 /Scale>
/Map>
...
/RaceCourseData>

2 - Create multiple <Course> tags in the XML file
3 - Create a GUI for the definition table

### Configuration
Mapper Version: 0.9.5
Operating System: Windows

**Comment by dl3sdo** [comment]
@Eric-P-Kmz: I added a commit that adds the map scale to the XML. It was also necessary to add the top left and bottom right positions to get an IOF compliant XML.
You can use one of the created artifacts from the Azure Pipeline to test the XML output with Vikazimut.

**Comment by Eric-P-Kmz** [comment]
Hello,

We have just tested this new version. No more errors. I'm preparing a video to show how to create a course with OOM and export it to Vikazimut.
Thanks

**Comment by dl3sdo** [comment]
Regarding topic 2 (multiple races in one XML):
You propose that it should be possible to create multiple, e.g., 'Simple Orienteering Course' objects and export them as multiple <RaceCourseData> elements of the same <Event> ?

**Comment by Eric-P-Kmz** [comment]
Yes, that's right. You'd have to be able to name 'Simple Orienteering Course' objects to do that .
 OCAD XML exemple
Course>
      Name>Chasse</Name>
      Length>2630</Length>
      Climb>0</Climb>
      CourseControl type="Start">
        <Control>S1</Control>
      </CourseControl>
      <CourseControl type="Control">
        <Control>31</Control>
        <LegLength>34</LegLength>
      </CourseControl>
      CourseControl type="Finish">
        Control>F1</Control>
        LegLength>63</LegLength>
      /CourseControl>
    /Course>

    Course>
      Name>Vert - facile</Name>
      Length>1300</Length>
      Climb>0</Climb>
      CourseControl type="Start">
        Control>S1</Control>
      /CourseControl>
      CourseControl type="Control">
        Control>31</Control>
        LegLength>34</LegLength>
      /CourseControl>
      CourseControl type="Finish">
        Control>F1</Control>
        LegLength>175</LegLength>
      /CourseControl>
    /Course>
  </RaceCourseData>
